source/configuration_manual/mail_location/obox/amazon_s3.rst

```diff
@@ -63,5 +63,8 @@ by adding the bucket region parameter to the S3 URL:
     obox_index_fs = compress:gz:6:s3: https://ACCESSKEY:SECRET@host/?bucket=BUCKETNAME&region=eu-central-1
   }
 
+The s3 driver also supports the bulk-deletion. The bulk-delete is enabled by default to delete up to 1000 key's with one request. To change this behaviour refer to ``bulk_delete_limit`` at :ref:`http_based_object_storages`
+
+  .. versionadded:: 2.3.10
 
 .. Warning:: AWS instances are known to react badly when high packets per second network traffic is generated by e.g. DNS lookups. Please see :ref:`os_configuration_dns_lookups`.
```

Comment:
you checked that this renders right?

Reply:
yes it renders right, the `versionadded` is intended to make it clearer that it just covers the previously mentioned topic.